netgrasp/netgrasp.py

```diff
@@ -904,7 +903,6 @@ def device_request(ip, mac):
         # Log request.
         ng.db.cursor.execute("SELECT request.rid, request.active FROM request WHERE request.ip = ? ORDER BY updated DESC LIMIT 1", (ip,))
         seen = ng.db.cursor.fetchone()
-        rid, active = (False, False)
```

Comment:
good catch; I was convinced we used these later but reading the code I see they get set before use in both code paths